fccodegen/_generator.py

```diff
@@ -1,26 +1,180 @@
-import fccodegen
+#
+# Functions related to generating model code.
+# TODO: Find a better name/description/layout
+#
+import fccodegen as cg
 import jinja2
 import os
+import sympy as sp
+import time
 
 
 def load_template(*name):
     """
     Returns a path to the given template
     """
-    path = os.path.join(fccodegen.DIR_TEMPLATE, *name)
+    path = os.path.join(cg.DIR_TEMPLATE, *name)
     #TODO: Check absolute path is still in DIR_TEMPLATE
     #TODO: Check exists
     #TODO: Look at the jinja PackageLoader. What's the advantage?
     with open(path, 'r') as f:
         return jinja2.Template(f.read())
 
 
+def load_model(path):
+    """
+    Loads a cellml model.
+    """
+    #TODO: This should become part of cellmlmanip
```

Comment:
Indeed :)

Reply:
@tamuri would you like me to document these somewhere, or shall we just do a review of fccodegen together some time and fix it on the fly?